Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1100,6 +1102,39 @@ namespace Babylon
             });
     }
 
+    void NativeEngine::LoadRawTexture(const Napi::CallbackInfo& info)
```

Comment:
Why don't we have `onSuccess`/`onError` callbacks for this one? It looks like all the other texture related loading functions have those.

Reply:
It's a JS synchronous operation https://github.com/BabylonJS/Babylon.js/blob/35ebb5c27d0cfbe42954a69a8a03843d1ca0a6e1/src/Engines/Extensions/engine.rawTexture.ts#L251

Reply:
If it is actually synchronous, we need to make the code actually synchronous. It's currently create a task that doesn't go anywhere. If an error occurs, it will go no where.